### DISCUSSION TITLE
Add Remixed youtube video https://youtu.be/BTB5DdKOIHw?si=1dzx08h6aCiaZCtd from https://www.youtube.com/watch?v=mJha4ls1MfM&list=PLebzuoh5ZI3LrVduRDqLcxCxlkruyDL27

### DISCUSSION THREAD

**Discussion by leechiuhui** [3dw/freemath#50] [Show and tell]
Add Remixed youtube video https://youtu.be/BTB5DdKOIHw?si=1dzx08h6aCiaZCtd from https://www.youtube.com/watch?v=mJha4ls1MfM&list=PLebzuoh5ZI3LrVduRDqLcxCxlkruyDL27
If this is not legal or suitable, or need any update,please don't hesitate to let me know. 
Thank you.

**Comment by bestian** [comment]
歡迎將Remix的版本加入自由數學的podcast

協作link我用email寄出

**Reply by leechiuhui** [reply]
> 歡迎將Remix的版本加入自由數學的podcast
> 
> 協作link我用email寄出

TTtttteacher, , the collaboration link you sent via email should give me editing permissions, right? But when I access the link, I can only view it, not collaborate.

**Comment by bestian** [comment]
確定已開放協作權限，
編輯清單方式請參考：

https://support.google.com/youtube/answer/57792?hl=zh-Hant&co=GENIE.Platform%3DDesktop


[image: 截圖 2024-06-28 凌晨4.42.25.png]


Be, a giver, not a taker
--Bestian Tang(唐宗浩)

=====

Bestian的網路工坊 <https://hack.bestian.tw> x 自由數學
<https://math.alearn.org.tw/#/> x 自主學習促進會 <http://www.alearn.org.tw/>

=====




Drago ***@***.***> 於 2024年6月27日 週四 下午11:50寫道：

**Comment by leechiuhui** [comment]
<img width="406" alt="截圖 2024-06-28 下午3 25 21" src="https://github.com/3dw/freemath/assets/107933615/48abfac1-30c2-406d-93ad-9faabd7e32a9">
Done,Thank you~:))

**Reply by leechiuhui** [reply]
1. The remixed version of the倒溯法is now online: https://youtu.be/_UvslKfYMh4?si=8pzWvFge7qIHs9v6

2. Collaborators of the YouTube Freemath Podcast cannot re-sort the playlist.

3. I anonymously posted in the "在家自學社群" and the "自學數學團"Not sure if it's legal.

**Reply by leechiuhui** [reply]
發現有1人按讚，有效果🥳

**Comment by leechiuhui** [comment]
https://youtu.be/5WlkHdOZ8_Q?si=jAXVZmBEZW5HALrM
自由數學簡介-自主學習促進會
